.github/workflows/cyclonus-netpol-test.yaml

```diff
@@ -7,27 +7,35 @@ on:
       - main
   pull_request:
     paths:
-      - 'npm/**'
-      - 'github/**'
-      - 'test/**'
+      - "npm/**"
+      - "github/**"
+      - "test/**"
   schedule:
     # run once a day at midnight
-    - cron: '0 0 * * *'
+    - cron: "0 0 * * *"
 
 jobs:
   cyclonus-test:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
         # run cyclonus tests in parallel for NPM with the given ConfigMaps
-        profile: [v1-default.yaml, v1-place-azure-chain-first.yaml, v2-default.yaml, v2-apply-on-need.yaml, v2-place-azure-after-kube-services.yaml]
+        profile:
+          [
+            v1-default.yaml,
```

Comment:
also working on this file for #1265, why don't we introduce fail fast for this file in that PR?